lib/srv/db/postgres/engine.go

```diff
@@ -388,6 +388,7 @@ func (e *Engine) getConnectConfig(ctx context.Context, sessionCtx *common.Sessio
 	// attempt fails. Reset the fallbacks to avoid retries, otherwise
 	// it's impossible to debug TLS connection errors.
 	config.Fallbacks = nil
+
```

Comment:
Nit: Need this change? :)

Reply:
Nope, leftover after the custom IP resolved. Removed.